Apps/frontend/src/views/StartStopButton.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+	import { createEventDispatcher } from 'svelte'
+
+	const dispatch = createEventDispatcher()
+
+	let play = "Start ▶";
+	let stop = "Stop ■";
+	let symbol = stop;
+	let hasStarted = true;
+
+	const handleStartStop = async () => {
+		hasStarted = !hasStarted;
+		hasStarted ? symbol = stop : symbol = play;	
+		dispatch('startStop', {buttonValue: hasStarted})
+	}
+	
+	
+</script>
+
+<button class="button-style button-up" on:click={handleStartStop}> 
```

Comment:
Funtionality works fine, only the styling could be improved (as @jenswaechtler already mentioned the text forces a different button size)
-> For me personally a simple button with the icons would be enough (as long as the size stays the same then)